mnefun/_mnefun.py

```diff
@@ -829,29 +873,128 @@ def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
             fnames_in.append(next_name)
         else:
             break
-    port = str(int(port))
-    t0 = time()
-    remote_ins = ['~/' + op.basename(fname) for fname in fnames_in]
-    remote_out = '~/temp_%s_raw_quat.fif' % t0
-    remote_hp = '~/temp_%s_hp.txt' % t0
+
     print('  Copying file to %s' % host)
-    cmd = ['scp', '-P' + port] + fnames_in + [host + ':~/']
+    cmd = ['scp', '-P' + str(port)] + fnames_in + [host + ':~/']
     run_subprocess(cmd, stdout=None, stderr=None)
+    t0 = time()
+    remote_ins = ['~/' + op.basename(f) for f in fnames_in]
+    if len(remote_ins) > 1:
+        fnames_out = [op.basename(r)[:-4] + '.tmp' for r in remote_ins]
+        assert len(fnames_out) == len(fnames_in)
+    else:
+        fnames_out = [op.basename(fname_out)[:-4] + '.tmp']
```

Comment:
do you need this conditional? It seems like it should work the same whether `len(remote_ins) == 1` or is `> 1`.